client/starwhale/core/dataset/type.py

```diff
@@ -389,6 +386,7 @@ def __init__(
         as_mask: bool = False,
         mask_uri: str = "",
         dtype: t.Type = numpy.uint8,
+        **kwargs: t.Any,
```

Comment:
In the `GrayscaleImage` subclass, we should use the explicit arguments, can we remove the kwargs argument? it will bring argument confusion in the calling phase.

Reply:
> In the `GrayscaleImage` subclass, we should use the explicit arguments, can we remove the kwargs argument? it will bring argument confusion in the calling phase.

 I don't want to duplicate the constructing parameters of `BaseArtifact` when initing subclass objects everywhere. `GrayscaleImage` is a subclass of `BaseArtifact`, is there a better way to propagate constructing parameters to `BaseArtifact`.

Reply:
**kwargs is replaced with link